Remotion/Validation/Core/MetaValidation/DefaultSystemPropertyMetaValidationRuleProvider.cs

```diff
@@ -16,15 +16,14 @@
 // 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using Remotion.Reflection;
 using Remotion.Utilities;
-using Remotion.Validation.MetaValidation.Rules.System;
 
 namespace Remotion.Validation.MetaValidation
 {
   /// <summary>
-  /// Default implementation of the <see cref="ISystemPropertyMetaValidationRuleProvider"/> interface. Provides a <see cref="MinimumLengthSystemPropertyMetaValidationRule"/> and
-  /// a <see cref="MaximumLengthSystemPropertyMetaValidationRule"/>.
+  /// Default implementation of the <see cref="ISystemPropertyMetaValidationRuleProvider"/> interface.
   /// </summary>
   public class DefaultSystemPropertyMetaValidationRuleProvider : ISystemPropertyMetaValidationRuleProvider
```

Comment:
Could you please create a ticket for dopping all the types up to and including ISystemPropertyMetaValidationRuleProviderFactory. The concept might be completely obsolete now. => I'm not 100% sure if we really should do this but we should have a ticket to either greenlight or set as won't fix.